components/odh-notebook-controller/controllers/suite_test.go

```diff
@@ -55,11 +55,14 @@ import (
 // +kubebuilder:docs-gen:collapse=Imports
 
 var (
-	cfg            *rest.Config
-	cli            client.Client
-	envTest        *envtest.Environment
+	cfg     *rest.Config
```

Comment:
The linter is right, but that's issue that has been in the repo for a long time already. See here for fix
* https://github.com/opendatahub-io/kubeflow/pull/462

Reply:
/override "Code static analysis / golangci-lint (components/odh-notebook-controller)"

Reply:
/override "Code static analysis / golangci-lint (components/odh-notebook-controller)"

Reply:
/override "golangci-lint (components/odh-notebook-controller)"

Reply:
yeah, I <s>rebased</s> merged